src/common/droid-util.c

```diff
@@ -233,11 +234,11 @@ pa_droid_profile_set *pa_droid_profile_set_new(const pa_droid_config_hw_module *
 
 static void add_default_profile(pa_droid_profile_set *ps,
                                 const pa_droid_config_hw_module *module,
-                                const pa_droid_config_output *primary_output,
-                                const pa_droid_config_output *low_latency_output,
-                                const pa_droid_config_output *media_latency_output,
-                                const pa_droid_config_input *builtin_input,
-                                const pa_droid_config_input *external_input,
+                                const pa_droid_config_device *primary_output,
+                                const pa_droid_config_device *low_latency_output,
+                                const pa_droid_config_device *media_latency_output,
+                                const pa_droid_config_device *builtin_input,
+                                const pa_droid_config_device *external_input,
```

Comment:
With different types, logical errors could be caught at compilation time ... just wondering if runtime assertions should be added now that it no longer is the case?